CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+### [2.508.1](https://github.com/kiva/ui/compare/v2.508.0...v2.508.1) (2023-03-01)
+
+
+### Bug Fixes
+
+* filters should be able to handle undefined facets object ([0fec7f8](https://github.com/kiva/ui/commit/0fec7f8acc1a4abb49593734b03864e89ef8c32e))
```

Comment:
You might consider merging `main` into `CIT-checkout-flow-improvements` branch directly first, so the PR just focuses on new changes.

Reply:
this would also work. I think the CIT checkout flow branch has enough changes for now though and this doesn't relate to checkout so it's good to keep separate.

Reply:
Yeah I'm not sure which branch is important here, like if it's a feature party branch.

Reply:
The reason CIT-checkout-flow-improvements isn't merged is because CIT has a visa campaign where we can't make any major feature changes to prod until the emails are sent out and the employees have made their loans.

Reply:
I don't have any intention to merge those changes in at least until after the March 8th campaign date.  @mcstover @dyersituations @tiarascoleman

Reply:
Makes sense @michelleinez thank you!